### PR TITLE
Conditonal rendering value

### PR DESCRIPTION
**Pull request by kwongz**
### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [x] I have added to the docs where applicable

Stories
image

Docs:
image

image

### CODE REVIEW

**Comment by hughess** [comment]
This looks great! A couple things to check:

- Is the red used for `redNegatives` the same as the one in `DataTable`?
- I would change the docs to match with the description of other color props in the product (there's a few in the `DataTable` docs)

**Comment by kwongz** [comment]
Updated red color to same in dataTable --> tableRow.svelte
image

@hughess in terms of the color props, the most similar prop that behaves the same as Value's would be the color prop in annotations.svelte

_annotations color prop:_
image
image

The Value Color Prop, can take in the same color options "options="CSS name | hexademical | RGB | HSL"

I'm also wondering if we should keep the title "**Color**" instead of "**Colour**" since the css and out prop is in American Spelling.

Let me know what you think and i can update the Value prop, and the title spelling in annotations if necessary

**Comment by kwongz** [comment]
@hughess I noticed Red Negative is listed in the data table docs, but missing from its prop list. Can we use Value's redNegatives prop list in Data Table

_Datatable proplist_
image

_Value prop list:_
image

**Comment by hughess** [comment]
Color is good for consistency

Ah good catch, yes you can use the same one as Value. But it should go under the `Column` section of the `DataTable` page.

Also looks like there might be a hanging closing tag on the DataTable page under the prop listing before `scaleColumn`

**Comment by kwongz** [comment]
Red negative added to column section in data tables
image

Removed /> above scaleColumn 
image

File: packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte [hunk 1]
```diff
@@ -101,7 +115,7 @@
 		>[{placeholder}]<span class="error-msg">Placeholder: no data currently referenced.</span></span
 	>
 {:else if !error}
-	<span>
+	<span class={fontColor} style="color: {fontColor}">
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte:118]: Why is `fontColor` being used as both a class and a style?

Comment (kwongz) [reply packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte:118]: thanks, originally we were trying to use tailwind styles, but then we decided to go with inline styling, will remove the class

**Comment by kwongz** [comment]
Value Docs:
image

DataTable Docs: 
image

**Comment by hughess** [comment]
Last thing - I think the rednegatives prop needs to be updated to accept strings for “true” and “false” for consistency with other components. The Column component rednegatives prop should already work like that.

goal with that is to remove the need for user to type curly braces to get Boolean values in the syntax - looks more approachable to data analyst, faster to type.

We should make this consistent across the product where Booleans are used in props

**Comment by zachstence** [comment]
> We should make this consistent across the product where Booleans are used in props

I've got this on my ToDo list for a general improvement to all core components

**Review by ItsMeBrianD — APPROVED** [review]
If the behavior here is good to go (@hughess @archiewood) then I think we're set, the code looks good to me

**Review by hughess — APPROVED** [review]
Works great!

**Pull request merged**